foundation/docs/sponsorship/friends.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Friends of PyTexas
+description: Enjoy what the PyTexas Foundation does and want to help support financially? We offer multiple tiers of support to choose from.
+---
+
+The PyTexas Foundation's mission is to build vibrant community of Python
+enthusiasts in the state of Texas. As PyTexas continues to grow, we 
+continuously rely on the generosity and support of individuals like you who 
```

Comment:
`continuously` might be a bit much. `rely` might be fine by itself, `rely on the continued ...` might give the sense of ongoing and reliable.

Reply:
Fixed